elasticsearch_dsl/document.py

```diff
@@ -150,6 +151,32 @@ def get(cls, id, using=None, index=None, **kwargs):
         return cls.from_es(doc)
 
     @classmethod
+    def mget(cls, ids, using=None, index=None, missing='none', **kwargs):
+        if missing not in frozenset(('raise', 'skip', 'none')):
+            raise ValueError("'missing' must be 'raise', 'skip', or 'none'.")
+        es = connections.get_connection(using or cls._doc_type.using)
+        index = index or cls._doc_type.index
+        docs = es.mget(
+            {'ids': ids},
+            index=index,
+            doc_type=cls._doc_type.name,
+            **kwargs
+        )
+        objs, missing_ids = [], []
+        for doc in docs['docs']:
+            if doc['found']:
+                objs.append(cls.from_es(doc))
```

Comment:
can we skip calling `from_es` if we know we will be raising `NotFoundError` anyway? it can be quite costly. Doing:
```
if doc['found'] and not missing_ids:
    objs.append(...)
elif not doc['found']:
    ....
```
Should do it